app/pdf.js

```diff
@@ -8,14 +8,23 @@ const electron = require('electron')
 const config = require('../js/constants/config')
 const BrowserWindow = electron.BrowserWindow
 
-const renderUrlToPdf = (appState, action, testingMode) => {
+const renderUrlToPdf = (appState, action) => {
   let url = action.url
   let savePath = action.savePath
   let openAfterwards = action.openAfterwards
 
   let currentBw = BrowserWindow.getFocusedWindow()
 
-  let bw = new BrowserWindow({show: !!testingMode, backgroundColor: '#ffffff'})
+  let bw = new BrowserWindow({
+    show: false,
```

Comment:
Just want to confirm: should this be false? (ex: do any tests use it). I remember seeing an issue where the PDF wasn't showing up... is that related? I think you chose to hide it because (on macOS), it opens a window on top of the print dialog, which I understand.

Reply:
they don't and this should be false. We only create this window to generate PDF which we want to save when window loads.